numpy/core/numeric.py

```diff
@@ -376,6 +376,153 @@ def extend_all(module):
             __all__.append(a)
 
 
+def count_nonzero(a, axis=None):
+    """
+    Counts the number of non-zero values in the array ``a``.
+    The word 'non-zero' is in reference to the Python 2.x
+    built-in method '__nonzero__()' (renamed '__bool__()'
```

Comment:
instead of single ticks, use double backticks for proper formatting. applies to this whole paragraph